.gitmodules

```diff
@@ -13,9 +13,11 @@
 [submodule "gomaxcompute"]
 	path = gomaxcompute
 	url = https://github.com/sql-machine-learning/gomaxcompute
+	branch = develop
 [submodule "gohive"]
 	path = gohive
 	url = https://github.com/sql-machine-learning/gohive
+	branch = develop
 [submodule "pysqlflow"]
 	path = pysqlflow
 	url = https://github.com/sql-machine-learning/pysqlflow
```

Comment:
Shall we add the `branch = develop` here?

Reply:
Done.